core/sys/info/cpu_intel.odin

```diff
@@ -67,7 +67,7 @@ init_cpu_features :: proc "c" () {
 	try_set(&set, .os_xsave,  27, ecx1)
 	try_set(&set, .rdrand,    30, ecx1)
 
-	when ODIN_OS == .FreeBSD || ODIN_OS == .OpenBSD {
+	when ODIN_OS == .FreeBSD || ODIN_OS == .OpenBSD || ODIN_OS == .NetBSD {
```

Comment:
Maybe edit the comment to indicate that this isn't allowed under netbsd as well.